pyjob/sge.py

```diff
@@ -104,13 +104,15 @@ def _create_runscript(self):
             cmd = '-q {}'.format(self.queue)
             runscript.append(self.__class__.SCRIPT_DIRECTIVE + ' ' + cmd)
         if self.runtime:
-            cmd = '-l h_rt={}'.format(self.runtime)
+            h, m = divmod(self.runtime, 60)
```

Comment:
given that you code duplicated this from PBS, you should really refactor this section to a function that accepts the runtime and returns the string. there should be tests and then we can ensure this behaves correctly.

Reply:
I agree, I will work on this.